crates/transaction-pool/src/blobstore/disk.rs

```diff
@@ -58,16 +58,27 @@ impl BlobStore for DiskFileBlobStore {
     }
 
     fn delete(&self, tx: B256) -> Result<(), BlobStoreError> {
-        self.inner.delete_one(tx)?;
+        let mut files_to_delete = self.inner.files_to_delete.write();
```

Comment:
prefer to keep the same style as `delete_all`

```
self.inner.files_to_delete.write().insert(txs);
```

Reply:
Done!